WFP-VAM/prism-app
# Generate pdf report using playwright; COUNTRY=cambodia;

This fixes issue #792.

This PR implements report generation in the backend with a 48h cache.

How to test the feature:

1. https://prism-900.surge.sh/?hazardLayerIds=flood_extent&date=2023-07-07
2. Flood extent -> Exposure Analysis -> Create Report 

screenshots: 

![image](https://github.com/WFP-VAM/prism-app/assets/100351051/4e1c72ee-5b4f-43dc-be51-544d63763228)

We give a shortcut with "Download report" to the user to avoid the preview and have a direct download. That way we can measure if users really need a preview. It also a mean to avoid triggering the slow react-pdf renderer for the user


Questions for @wadhwamatic:
- Currently, Preview Report still uses the frontend generation and is quite slow. Now that download report is faster, should we remove the preview button?
- Is 48h ok for the cache?
